services/gitclub/app/routes/helpers.py

```diff
@@ -1,24 +1,97 @@
 from os import getenv
-from typing import Any, List, Type
+from typing import Any, List, Optional, Type
 
 from flask import g
+from flask_caching import Cache
 from oso_cloud import Oso
 from sqlalchemy.orm.session import Session
-from werkzeug.exceptions import Forbidden, NotFound
+from sqlalchemy.future import select
+from werkzeug.exceptions import Forbidden, NotFound, Unauthorized
+
+from app.models import Repository, Organization, Issue, OrgRole, RepoRole, User
 
 oso = Oso(url=getenv("OSO_URL", "https://cloud.osohq.com"), api_key=getenv("OSO_AUTH"))
+cache = Cache(config={'CACHE_TYPE': 'SimpleCache'})
+
+def object_to_typed_id(obj: Any, allow_unbound=False) -> dict:
+    if isinstance(obj, str):
+        return {"type": "String", "id": obj}
+    elif isinstance(obj, dict):
+        assert allow_unbound or ("type" in obj and "id" in obj)
+        if "type" in obj:
+            obj["type"] = str(obj["type"])
+        if "id" in obj:
+            obj["id"] = str(obj["id"])
+        return obj
+    elif isinstance(obj, User):
+        return { "type": "User", "id": obj.username}
+    elif obj is None:
+        return {}
+    else:
+        return { "type": obj.__class__.__name__, "id": str(obj.id) }
+
+def current_user():
+    if g.current_user is None:
+        raise Unauthorized
+    return object_to_typed_id(g.current_user)
 
 
+def tell(predicate: str, *args: Any):
+    print(f'oso-cloud tell {predicate} {",".join([str(a) for a in args])}')
+    return oso.tell(predicate, *[object_to_typed_id(a, True) for a in args])
```

Comment:
```suggestion
    return oso.tell(predicate, *[object_to_typed_id(a) for a in args])
```
I don't think we want to allow unbound things in `tell` commands.